src/profile/external-identities/OAuth2Providers.test.ts

```diff
@@ -0,0 +1,12 @@
+import { oauth2Provider } from 'src/profile/external-identities/OAuth2Providers';
+
+describe('OAuth2Provider', () => {
+  it('should return the correct provider for GitHub', () => {
```

Comment:
Not a big deal to repeat for two providers, but if more are to be added, this might be a good place for `it.each`
https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Honestly, going more for test coverage metrics than anything else here...